src/pages/DashboardPage.jsx

```diff
@@ -1,14 +1,43 @@
 import Table from "../components/Table";
-import { columns, rows } from "../common/dashboardConfig";
+import { columnsIncome, columnsExpense } from "../common/dashboardConfig";
+import { GetData } from "../services/ExpenseGroupService";
+import { FetchData } from "../services/IncomeGroupService";
+import { useQuery } from "react-query";
 
 const Dashboard = () => {
+  const URLexpense = "http://localhost:4000/expense/last-five";
+  const URLincome = "http://localhost:4000/income/last-five";
+
+  const { isLoading, isError, data, error } = useQuery(
+    ["todos", URLexpense],
```

Comment:
Try to use a descriptive key, it is not todos in our case but it can be maybe lastFiveExpenses